net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/MetadataFactory.java

```diff
@@ -253,7 +253,7 @@ public static RuleMetadata createRuleMetadata(ModuleDetails moduleDetails) {
     }
 
     RuleMetadata ruleMeta = new RuleMetadata(moduleDetails.getName(), moduleDetails.getName(),
-            moduleDetails.getParent(), MetadataFactory.getDefaultSeverity(),
+            moduleDetails.getParent().split("\\.")[4], MetadataFactory.getDefaultSeverity(),
```

Comment:
This would not survive a package depth changes. You should at least do 
```
String[] splitParent = moduleDetails.getParent().split("\\.");
splitParent[splitParent.length - 1]
```

Reply:
Sorry, forgot to change, done now.